src/web/js/dashboard/GoogleAPI.js

```diff
@@ -14,48 +14,58 @@ class GoogleAPI {
   /**
    *  Sign in the user upon button click.
    */
-  signIn = (event) => {
-    return gwrap.load({name: 'drive',
+  signIn = () => {
+    return window.gwrap.load({name: 'drive',
       version: 'v3',
       reauth: {
         immediate: false
       }
     });
   }
 
-  /**
-   *  Sign out the user upon button click.
-   */
-  signOut = (event) => {
-    throw "Can't sign out yet.";
-  }
+  removeAllAppDataFiles = () => {
+     return window.gapi.client.drive.files.list({
+       spaces: 'appDataFolder'
+     }).then(resp => {
+       resp.result.files.forEach(file => {
+         window.gapi.client.drive.files.delete({'fileId': file.id}).then(console.log);
+       });
+     });
+   }
 
-  createAppFolder = (appName) => {
+  createAppFolder = () => {
     return window.gapi.client.drive.files.create({
       resource: {
-        'name' : appName,
+        'name' : 'pyret',
         'mimeType' : 'application/vnd.google-apps.folder'
       }
     });
   }
 
-  getAppFolderID = (appName) => {
+  // for use while testing
+  removeFileOrFolder = (id) => {
+    return window.gapi.client.drive.files.delete({
+      'fileId': id
+    });
+  }
+
+  // ACTUAL FUNCTION: lists all files in appDataFolder with name = 'pyret'
+  getAppFolderID = () => {
```

Comment:
I'm pretty sure you're right. I wrote this comment while I was initially trying to understand the code in this file and I was incorrect in my assumption of how this one works. The comment can just be deleted.